src/subscription_manager/utils.py

```diff
@@ -62,6 +62,76 @@ def __repr__(self):
         return pprint.pformat(self.as_dict())
 
 
+def three_way_merge(local=None, base=None, remote=None, on_conflict="remote"):
+    """
+    Performs a three-way merge on the local and remote dictionaries with a given base.
+    :param local: The dictionary of the current local values
+    :param base: The dictionary with the values we've last seen
+    :param remote: The dictionary with "their" values
+    :param winner_on_conflict: Either "remote" or "local" or None. If "remote", the remote changes
+                               will win any conflict. If "local", the local changes will win any
+                               conflict. If anything else, an error will be thrown.
```

Comment:
I cannot image any case, where `on_conflict` is something else then "local" or "remote". Please, can you add any example here?

Reply:
There is no valid case where it is expected to be anything else.